doc/elements.md

```diff
@@ -187,6 +187,32 @@ A = np.array([[1.23], [4.56]])
 If a variable `v` is a complex object, you should use `import prairielearn as pl` and `data['params'][params_name] = pl.to_json(v)`.
 
 
+## `pl-latex-matrix-display` element
```

Comment:
What do you think about the alternative name `pl-matrix-latex`? The idea would be that we've got elements that do things with 2d numpy arrays (or scalars) - input, output, or in this case create a latex string.